docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+The selectorTerms specified inside the *namespaceOffloading* are applied on *every pod* created inside the namespace.
+
+If not specified at creation time, the **clusterSelector** will target all virtual nodes available, enabling the offloading on all peered cluster. 
+More precisely, the default value corresponds to:
+
+```yaml
+clusterSelector:
+  nodeSelectorTerms:
+  - matchExpressions:
+    - key: liqo.io/type
+      operator: In
+      values:
+      - virtual-node
+``` 
+
+{{% notice info %}}
+In **Liqo 0.3**, the *namespaceOffloading* must contain the configuration at creation time. 
+If you want to modify its structure at runtime, you should delete the resource and recreate it with a new configuration. 
+This triggers the deletion of all remote "twin" namespaces and the creation of the new ones.
+{{% /notice %}}
+
+The *matchExpressions*, specified in the **clusterSelector** term, select labels attached to Liqo virtual nodes.
+
+More precisely, at Liqo installation time, you may identify a set of labels to expose the most relevant features of your cluster. 
+After the peering phase, the virtual node will expose those labels, enabling the possibility to select it during the offloading configuration. 
+Moreover, the scheduler will also be able to use these virtual nodes' labels to impose affinity/anti-affinity policies at run-time.
+
+It is worth noting that there is no restriction on the labels to choose. 
+Labels can characterize your clusters showing their geographical location, the underlying provider, or the presence of specific hardware devices.
+
+{{% notice tip %}}
+If you just want to create deployment topologies that include all available clusters you are not required to choose labels at installation time. 
+All virtual nodes expose the label `liqo.io/type = virtual-node` by default.
+{{% /notice %}}
+
+### NamespaceOffloading in quick offloading
+
+Also the *quick Offloading* relies on the namespaceOflloading resource. 
+In fact, when the label `liqo.io/enabled = true` is added to a namespace, this event triggers the creation of a default *namespaceOffloading* resource inside the namespace.
+
+The generated resource is exactly equal to the template [seen above](#custom-offloading), setting all fields to the default values:
+
+* **namespaceMappingStrategy**: *DefaultName*.
+* **podOffloadingStrategy**: *LocalAndRemote*.
+* **clusterSelector**: *all remote clusters selection*.
+
+{{% notice info %}}
+Using the *quick offloading*, it is not possible to customize the generated *namespaceOffloading* resource. 
+If a *namespaceOffloading* is already present in the namespace, you should remove the label first and then create the new resource.
+Removing the label is equivalent to removing the resource directly.
```

Comment:
Isn't it added back by the operator if the label is present and the resource is not present?

Reply:
ATM no, it is not recreated the resource. We will fix this aspect, thank you.